src/use-lunatic/reducer/reduce-go-next-page.ts

```diff
@@ -42,4 +42,6 @@ function reduceGoNextPage(state: LunaticState): LunaticState {
 	};
 }
 
-export default compose<ActionGoNextPage>(reduceGoNextPage, overviewOnChange);
+export default function (state: LunaticState) {
```

Comment:
maybe it's the good moment to remove default export and make changes where function is imported

Reply:
Done